content/docs/1.1.1/deploy/install/_index.md

```diff
@@ -106,6 +106,14 @@ For RHEL, CentOS, and EKS with `EKS Kubernetes Worker AMI with AmazonLinux2 imag
 
 ```
 yum install iscsi-initiator-utils
+echo "InitiatorName=$(/sbin/iscsi-iname)" > /etc/iscsi/initiatorname.iscsi
+```
+
+Make sure the iscsid service is enabled and started after open-iscsi installed.
```

Comment:
```suggestion
Make sure the `iscsid` service is enabled and started after `iscsi-initiator-utils` is installed.
```

Reply:
Use **iSCSI tool package** instead of `iscsi-initiator-utils` for all Linux distributions?
@innobead WDYT?

Reply:
I was misled by different distro parts. Yeah, sounds good to me.